lib/anvil.ex

```diff
@@ -1,2 +1,63 @@
 defmodule Anvil do
+  defmodule UndefinedFixture do
+    defexception [:message]
+
+    def exception(fixture_name) do
+      message = "No fixture defined for #{inspect fixture_name}"
+      %UndefinedFixture{message: message}
+    end
+  end
+
+  defmacro __using__(_opts) do
+    quote do
+      @before_compile unquote(__MODULE__)
+
+      def assoc(opts, fixture_name) do
```

Comment:
I think that is a great idea, but I think the problem is that it doesn't actually define the functions. It only import them. We need to define the functions so you can call `create`, `build`, etc. 

Instead I can define the function in the using macro and then call function in `Anvil`. I think that is a good balance between the two